source/_posts/generating-pdf-in-ruby-on-rails.md

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
use keyword throught the article, which is not seen right now

Reply:
What does this mean?

Reply:
I can see some keywords used in the blog

Reply:
it should be placed in there four places if possible, i am finding only in top in the content